source/particles/particle_handler.cc

```diff
@@ -69,12 +69,13 @@ namespace Particles
   template <int dim, int spacedim>
   void
   ParticleHandler<dim, spacedim>::initialize(
-    const parallel::distributed::Triangulation<dim, spacedim> &tria,
-    const Mapping<dim, spacedim> &                             mapp,
-    const unsigned int                                         n_properties)
+    const parallel::distributed::Triangulation<dim, spacedim>
+      &                           new_triangulation,
```

Comment:
Do we really want such a long name?

Reply:
It's in our internal cc file so I'd say it's fine :-)